<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="menu">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>461</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="ex_1">
         <property name="text">
          <string>задание №1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ex_2">
         <property name="text">
          <string>задание №2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ex_3">
         <property name="text">
          <string>задание №3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="exersize">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>760</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>51</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="exersize_1">
       <widget class="QSpinBox" name="N_1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QPushButton" name="start_1">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>10</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>N = </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="exersize_2">
       <widget class="QDoubleSpinBox" name="P_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>65</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
       <widget class="QPushButton" name="start_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>89</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>10</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>min = </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="max_2">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>30</y>
          <width>110</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="entro_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>10</y>
          <width>110</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>10</y>
          <width>31</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>H = </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="min_2">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>10</y>
          <width>110</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>30</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>max = </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="exersize_3">
       <widget class="QSpinBox" name="N_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QPushButton" name="start_31">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>30</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>max = </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="max_3">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>30</y>
          <width>110</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="entro_3">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>10</y>
          <width>110</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>10</y>
          <width>30</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>H = </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>N = </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>M = </string>
        </property>
       </widget>
       <widget class="QSpinBox" name="M_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>K = </string>
        </property>
       </widget>
       <widget class="QSpinBox" name="K_3">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>20</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QPushButton" name="start_32">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>20</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="to_menu">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>570</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Вернуться</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>159</y>
        <width>760</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
